services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/NbWorkerTopology.java

```diff
@@ -73,6 +82,12 @@
     private static final String DISCO_KAFKA_BOLT_NAME = "disco-kafka-bolt";
     private static final String HISTORY_BOLT_NAME = "history-operations-bolt";
     private static final String NB_SPOUT_ID = "nb-spout";
+    private static final String SPEAKER_KAFKA_BOLT = "speaker-bolt";
+
+    private static final int HUB_TIMEOUT_MS = 13000;
```

Comment:
please add comment here or update pr as discussed

Reply:
This has been moved to NbWorkerTopologyConfig.